io.edgehog.devicemanager.apps.AvailableNetworks.json

```diff
@@ -0,0 +1,39 @@
+{
+  "interface_name": "io.edgehog.devicemanager.apps.AvailableNetworks",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "properties",
+  "ownership": "device",
+  "mappings": [
+    {
+      "allow_unset": true,
+      "endpoint": "/%{network_id}/id",
+      "type": "string",
+      "description": "Network id"
+    },
+    {
+      "allow_unset": true,
+      "endpoint": "/%{network_id}/driver",
+      "type": "string",
+      "description": "Network driver"
+    },
+    {
+      "allow_unset": true,
+      "endpoint": "/%{network_id}/checkDuplicate",
+      "type": "boolean",
+      "description": "Network check duplicates"
```

Comment:
Might be wrong as I'm unsure what this does, but doesn't "Duplicate network check" sound better?

Reply:
or "Check duplicate networks" as you have better written elsewhere in this PR